src/pages/login/nl.js

```diff
@@ -0,0 +1,19 @@
+import { useCookies } from 'react-cookie';
+import { LoginButton } from '../../components/LoginButton';
+
+const Nl = () => {
+  const cookieOptions = {
+    path: '/',
+    maxAge: 60 * 60 * 24 * 30,
+  };
+  const [, setCookie] = useCookies(['udb-language']);
+  setCookie('udb-language', 'nl', cookieOptions);
+
+  return (
+    <div>
+      <LoginButton language="nl" />
+    </div>
+  );
+};
+
+export default Nl;
```

Comment:
nl and fr should be the same page but with different copy, can't we solve this by using a [...params].js file?